packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -42,11 +42,16 @@ function itemFilter(inputValue: string) {
 }
 
 function estimateSize() {
-  return 60;
+  return 45;
 }
 
-export const Combobox = ({ options, onChange, value, ...restProps }: ComboboxProps) => {
-  const MIN_WIDTH = 400;
+const MIN_HEIGHT = 400;
+// On every 100th index we will recalculate the width of the popover.
+const INDEX_WIDTH_CALCULATION = 100;
+// A multiplier guesstimate times the amount of characters. I any padding or image support etc. is added this will need to be updated.
```

Comment:
```suggestion
// A multiplier guesstimate times the amount of characters. If any padding or image support etc. is added this will need to be updated.
```
Great adding a comment 👏